src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -87,15 +87,30 @@
     </table>
 
     {% if collector.logs %}
-        <ul class="alt">
-            {% for log in collector.logs if priority >= 0 and log.priority >= priority or priority < 0 and log.context.type|default(0) == priority %}
-                <li class="{{ cycle(['odd', 'even'], loop.index) }}{% if log.priority >= 400 %} error{% elseif log.priority >= 300 %} warning{% endif %}{% if log.context.scream is defined %} scream{% endif %}">
-                    {{ logger.display_message(loop.index, log) }}
-                </li>
+        <table>
+            <tr>
+                <th>#</th>
+                <th>Priority</th>
+                <th>Channel</th>
+                <th>Message and context</th>
+            </tr>
+            {% for i, log in collector.logs if priority >= 0 and log.priority >= priority or priority < 0 and log.context.type|default(0) == priority %}
+                <tr class="{% if log.priority >= 400 %}error{% elseif log.priority >= 300 %}warning{% endif %}{% if log.context.scream is defined %}scream{% endif %}">
+                    <td>{{ i + 1 }}</td>
+                    <td>
+                        {%- if log.context.type is defined and (constant('E_DEPRECATED') == log.context.type or constant('E_USER_DEPRECATED') == log.context.type) -%}
+                            DEPRECATION
+                        {%- else -%}
+                            {{ log.priorityName }}
+                        {%- endif -%}
+                    </td>
+                    <td>{{ log.channel is defined ? log.channel }}</td>
```

Comment:
What about removing the column when we don't have information about the channel?